README.md

```diff
@@ -36,4 +36,32 @@ If you want to run the smoketests locally, you can use the following commands:
 ```bash
 docker-compose run smoketests
 ```
-You might need to flush your redis database before running the tests. As it assumes that the database is empty.
\ No newline at end of file
+You might need to flush your redis database before running the tests. As it assumes that the database is empty.
+
+## How to make changes to the SQL database schema
+
+Making a change to the database schema in `models.py` will require a database migration.
+
+If you want to update the models, first make the change in `models.py` and then
+prepare and run a migration.  You should ensure that any model change is committed
+along with its migration so that the two do not get out of sync.
+
+Once you have made your `models.py` change:
+
+1. Prepare your migration with:
+   ```bash
+   docker-compose run web alembic revision --autogenerate -m "Title of migration"
+   ``` 
+   If the command ran successfully, a new file will be created under `alembic/versions`.
+
+2. Run the migration after the file has been created via
+   ```bash
+   docker-compose run web alembic upgrade head
```

Comment:
giving an example for downgrading might be helpful for dev at some points? but I guess that's pretty trivial anyway.

Reply:
Knowing how to reverse a bad migration (for example if you typo'd a model) might be useful yes.  Is that hard to give an example for Adam?

Reply:
I have since expanded the readme. I think it's OK, but have I covered enough?

Reply:
I believe so, I'll resolve this.